cypress/e2e/shared/telegrafPlugins.test.ts

```diff
@@ -133,6 +133,13 @@ describe('Sources > Telegraf Plugins', () => {
       cy.getByTestID('next').click()
       cy.getByTestID('overlay--mask').should('not.exist')
 
+      // go back to the sources page
```

Comment:
New flow goes to the telegraf page after we save a config so we need to be back at the sources page, for this test.